src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+  width: 200px;
+  padding: 16px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  justify-content: space-between;
+
+  &:nth-child(n) {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  // &:nth-last-child(-n + 4) {
+  //   margin-bottom: 0;
+  // }
```

Comment:
position, width, margins are external styles
<img width="992" alt="image" src="https://user-images.githubusercontent.com/102411130/200793915-93d03aa3-08e1-431d-81f2-6c213f9a747b.png">

fix all such cases and also check css properties order http://codeguide.academy/html-css.html#css-order